test/tools/libtesteth/ImportTest.h

```diff
@@ -69,10 +69,13 @@ class ImportTest
 	eth::State m_statePost;
 
 private:
-	using ExecOutput = std::pair<eth::ExecutionResult, eth::TransactionReceipt>;
+    struct transactionToExecute;
```

Comment:
This is not needed, it's defined in the scope of the same class.

Reply:
compiler errors on declaration of this method 
```
bool findExpectSectionForTransaction(
        transactionToExecute const& _tr, eth::Network const& _net, bool _isFilling);

```

Reply:
Oh ok, I was wrong.
I would move then this struct declaration to here, above all methods.